src/RollerConveyor/Roller.cs

```diff
@@ -42,7 +42,10 @@ public override void _Ready()
 	{
 		meshInstance = GetNode<MeshInstance3D>("MeshInstance3D");
 		staticBody = GetNode<StaticBody3D>("StaticBody3D");
-		material = meshInstance.Mesh.SurfaceGetMaterial(0) as StandardMaterial3D;
+		Mesh mesh = meshInstance.Mesh.Duplicate() as Mesh;
```

Comment:
Was working on the same thing per the last commit and breaking things after the one before that.

Why not just use "Local to Scene" for the roller?

![image](https://github.com/user-attachments/assets/ddd26d67-8ab0-4962-93a3-73f7e0b5783d)

Reply:
I think that would still share it since they're all the same instance. I was just doing a technique I've seen a lot of the other parts use.

Though I'm not fond of it. Every load generates a new resources which dirties the git diffs for scenes.

Reply:
>I think that would still share it since they're all the same instance. I was just doing a technique I've seen a lot of the other parts use.

Seemed to work for me, but yeah, you are right duplicate was used for other parts. Currently ambivalent about which method to use.  

>Though I'm not fond of it. Every load generates a new resources which dirties the git diffs for scenes.

Yeah... that is unfortunate.